src/pages/settings/Wallet/WalletPage/WalletPage.js

```diff
@@ -106,7 +105,7 @@ function WalletPage({bankAccountList, betas, cardList, fundList, isLoadingPaymen
     }, [shouldShowEmptyState, windowWidth]);
 
     const getSelectedPaymentMethodID = useCallback(() => {
-        if (paymentMethod.selectedPaymentMethodType === CONST.PAYMENT_METHODS.PERSONAL_BANK_ACCOUNT) {
+        if (paymentMethod.selectedPaymentMethodType === CONST.PAYMENT_METHODS.BANK_ACCOUNT) {
```

Comment:
Were all this changes on purpose? Oh, the old constant did not even exist?? 😕

Reply:
Please undo these changes. It was renamed in https://github.com/Expensify/App/pull/30582

Reply:
My bad, good catch, not sure what happened here, I did not manually change these 😕 maybe because I used a git stash at some point 😕 Fixed